examples/framework/preferences/usingGrades/index.html

```diff
@@ -64,6 +64,8 @@
     <body class="prefsEditor-example-theme fl-focus">
         <!-- The page-wide UI Enhancer needs to be initialized right away -->
         <script type="text/javascript">
+            // FLUID-6300 - this page does NOT use the fluid.uiOptions.prefsEditor.localized
```

Comment:
I think you filed a different JIRA to add the localization support the grade versions of the prefs framework. You should use that JIRA number here instead.